src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java

```diff
@@ -0,0 +1,145 @@
+package com.github.wovnio.wovnjava;
+
+import java.lang.StringBuilder;
+import java.util.ArrayList;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Comment;
+import org.jsoup.nodes.Element;
+import org.jsoup.parser.Tag;
+import org.jsoup.select.Elements;
+
+class HtmlConverter {
+    private final ArrayList<String> markers = new ArrayList<String>();
+    private final String prefix = "wovn-marker-";
```

Comment:
could you please change prefix to CAPITAL and have a better name? like WOVN_MARKER_PREFIX or something else